### DISCUSSION TITLE
Filter values

### DISCUSSION THREAD

**Discussion by Ulee777** [microsoft-search/pnp-modern-search#2271] [Q&A] [answered]
Hello, 

I am struggling with Filter for People Search.  I am trying to use Managed property "Manager" in the filter. 
I am getting i:0#.f|membership|user@email.com as values. 
Could someone please direct me on how to hide "i:0#.f|membership" from the values?

**Comment by FranckyC** [comment, marked as answer]
Hi @Ulee777, for this, you can use the Handlebars helper `getUserEmail` to only get email in your template like this:

```
{{getUserEmail value}}
```

**Comment by 4iJacob** [comment]
Hi can someone give an example for an adaptive card. The managed property returns
i:0#.f|membership|user@email.com. How do I get the displayname or just the first part of the email address (before @)

**Reply by wobba** [reply]
If you see the sample at https://microsoft-search.github.io/pnp-modern-search/extensibility/adaptivecards_customizations/ you can use handlebar helper functions inside the text nodes and do manipulation. You can also see if any of the adaptive card pre-build functions work, but I haven't been able to find clear docs on what works for the SPFx adaptive card renderer.
